index.html

```diff
@@ -13,16 +13,34 @@
 
     <script>
         var calculator = {
-            addNumbers: function () {
+            addNumbers: function (numberOne, numberTwo) {
```

Comment:
Having the parameters defined on line 16 is not needed. We do not take input into the function. Instead we get the values to work with by taking them from values in the form fields.  Lines 18 and 19 for example are getting the values we will use.